### PR TITLE
Implemented Static UI for Find Order page

### PR DESCRIPTION
**Pull request by Utkarshkaraiya**

### CODE REVIEW

**Comment by ymaheshwari1** [comment]
Please add branch id after each commit message and PR title

File: src/views/Order.vue [hunk 1]
```diff
@@ -1,118 +1,283 @@
...
-        <div class="order-tags">
-          <ion-chip>
-            <ion-icon/>
-            <ion-label>
-              Shopify-Id
-            </ion-label>
-          </ion-chip>
+    <ion-content>
+      <div class="order">
+    <div class="search">
```
Comment (ymaheshwari1) [inline src/views/Order.vue]: Improve indentation

File: src/views/Order.vue [hunk 2]
```diff
@@ -1,118 +1,283 @@
...
+         <ion-card-content>
            <ion-chip>
-            <ion-icon/>
-            <ion-label>
-             Customer Loyalty Status
-            </ion-label>
-          </ion-chip>
+             <ion-label>bhdb</ion-label>
+           </ion-chip>
+             <ion-chip>
```
Comment (ymaheshwari1) [inline src/views/Order.vue]: Improve indentation

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/locales/en.json [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
   "Ready to create an app?": "Ready to create an app?",
   "Settings":"Settings",
+  "Size":"Size",
   "Start with Ionic": "Start with Ionic",
   "Store":"Store",
   "store name":"store name",
-  "UI Components": "UI Components"
+  "UI Components":"UI Components"
+
+
```
Comment (ymaheshwari1) [inline src/locales/en.json]: Remove unwanted lines

File: src/views/Order.vue [hunk 4]
```diff
@@ -1,118 +1,279 @@
...
+                </ion-thumbnail>
+                <ion-label>
+                  <p>Brand</p>
+                  <h1>Virtual Name</h1>
+                  <p>{{ $t("Color") }} : color</p>
+                  <p>{{ $t("Size") }}: size</p>
+                </ion-label>
+                <ion-note slot="end" color="success">
+                  {{ $t("In Stock", { count: 15 }) }}</ion-note
+                >
```
Comment (ymaheshwari1) [inline src/views/Order.vue]: Properly close the tags.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Order.vue (hunk 1 above)
Comment (Utkarshkaraiya) [reply src/views/Order.vue]: Done sir

File: src/views/Order.vue (hunk 2 above)
Comment (Utkarshkaraiya) [reply src/views/Order.vue]: Done sir

File: src/locales/en.json (hunk 3 above)
Comment (Utkarshkaraiya) [reply src/locales/en.json]: Done sir

File: src/views/Order.vue (hunk 4 above)
Comment (Utkarshkaraiya) [reply src/views/Order.vue]: Done sir

**Review by ymaheshwari1 — APPROVED** [review]
LGTM

**Pull request merged**